src/SFML/Graphics/RenderTarget.cpp

```diff
@@ -147,6 +147,50 @@ std::uint32_t equationToGlConstant(sf::BlendMode::Equation blendEquation)
 
     return GLEXT_GL_FUNC_ADD;
 }
+
+
+// Convert an UpdateOperation constant to the corresponding OpenGL constant.
+std::uint32_t stencilOperationToGlConstant(sf::StencilUpdateOperation operation)
+{
+    // clang-format off
+    switch (operation)
+    {
+        case sf::StencilUpdateOperation::Keep:      return GL_KEEP;
+        case sf::StencilUpdateOperation::Zero:      return GL_ZERO;
+        case sf::StencilUpdateOperation::Replace:   return GL_REPLACE;
+        case sf::StencilUpdateOperation::Increment: return GL_INCR;
+        case sf::StencilUpdateOperation::Decrement: return GL_INCR;
+        case sf::StencilUpdateOperation::Invert:    return GL_INVERT;
+    }
+    // clang-format on
+
+    sf::err() << "Invalid value for sf::StencilUpdateOperation! Fallback to sf::StencilMode::Keep." << std::endl;
+    assert(false);
+    return GL_KEEP;
+}
+
+
+// Convert a Comparison constant to the corresponding OpenGL constant.
+std::uint32_t stencilFunctionToGlConstant(sf::StencilComparison comparison)
+{
+    // clang-format off
+    switch (comparison)
+    {
+        case sf::StencilComparison::Never:        return GL_NEVER;
+        case sf::StencilComparison::Less:         return GL_LESS;
+        case sf::StencilComparison::LessEqual:    return GL_LEQUAL;
+        case sf::StencilComparison::Greater:      return GL_GREATER;
+        case sf::StencilComparison::GreaterEqual: return GL_GEQUAL;
+        case sf::StencilComparison::Equal:        return GL_EQUAL;
+        case sf::StencilComparison::NotEqual:     return GL_NOTEQUAL;
+        case sf::StencilComparison::Always:       return GL_ALWAYS;
+    }
+    // clang-format on
+
+    sf::err() << "Invalid value for sf::StencilComparison! Fallback to sf::StencilMode::Always." << std::endl;
+    assert(false);
```

Comment:
Elsewhere we've kept `assert(false)` when there's an `sf::err()` log message right above it since otherwise we'd just be writing the same string twice